### PR TITLE
update dependencies and snowball test 

### PR DESCRIPTION
**Pull request by jasdel**
Update integration test module dependencies, and fix snowball test.

### CODE REVIEW

File: internal/integrationtest/go.mod [hunk 1]
```diff
@@ -2,53 +2,52 @@ module github.com/aws/aws-sdk-go-v2/internal/integrationtest
...
+	github.com/aws/aws-sdk-go-v2/service/sqs v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/ssm v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/sts v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/support v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/waf v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/wafregional v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/wafv2 v0.0.0-20200925183546-de4a17a19ce2
+	github.com/aws/aws-sdk-go-v2/service/workspaces v0.0.0-20200925183546-de4a17a19ce2
 	github.com/awslabs/smithy-go v0.0.0-20200924210334-28773c6e7960
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
```
Comment (skotambkar) [inline internal/integrationtest/go.mod:50]: why do we need this?

**Review by skotambkar — APPROVED** [review]
Just a comment on

File: internal/integrationtest/go.mod (hunk 1 above)
Comment (jasdel) [reply internal/integrationtest/go.mod:50]: not sure go tidy added it.

Comment (jasdel) [reply internal/integrationtest/go.mod:50]: indirect dep from go-cmp:

```
# golang.org/x/xerrors
github.com/aws/aws-sdk-go-v2/internal/integrationtest
github.com/aws/aws-sdk-go-v2/config
github.com/aws/aws-sdk-go-v2/ec2imds
github.com/aws/aws-sdk-go-v2/aws/middleware
github.com/aws/aws-sdk-go-v2/aws/middleware.test
github.com/google/go-cmp/cmp/cmpopts
golang.org/x/xerrors
```

**Pull request merged**